src/generators/orama-db/constants.mjs

```diff
@@ -0,0 +1,7 @@
+// This schema comes from
+// https://github.com/oramasearch/orama-ui-components/blob/main/packages/ui-stencil/src/types/index.ts#L4
+export const SCHEMA = {
+  title: 'string',
+  description: 'string',
+  path: 'string',
+};
```

Comment:
Hi!
This schema is the one used to display the results. You can add other properties that Orama will search for. This will help add metadata like tags and similar to all documents. 
```
export const SCHEMA = {
  title: 'string',
  description: 'string',
  path: 'string',
  tags: 'string[]', // or 'string'
};
```
@ovflowd WDYT about this enhancement?

NB: You can also play with the "boost" parameter. From my experience, boosting the title and similar fields is helpful because people commonly want to see first the title-matching result and the rest later.

Reply:
cc @avivkeller; Thank you, @allevo!

Reply:
Sounds good to me, but we'd need to add tags / whatnot to the upstream docs, or hardcode them here. I feel like, while more work, upstreaming the tags aligns more with #332.